can-auth/vatican.c

```diff
@@ -212,34 +259,90 @@ int VULCAN_FUNC vulcan_send(ican_t *ican, uint16_t id, uint8_t *buf,
     // NOTE: do not block for ACK, such that we can start the MAC computation
     rv = vatican_send(ican, id, buf, len, /*block=*/0);
 
+    #if VATITACAN
+            // Delay authentication message
+            sleeping = 0x1;
```

Comment:
it will be easier and more secure to do busy waiting w NOP instructions or something in the enclave itself? eg
https://github.com/sancus-pma/vulcan/blob/master/drivers/mcp2515.c#L453